src/Module/Contact/Profile.php

```diff
@@ -340,13 +340,8 @@ protected function content(array $request = []): string
 			];
 		}
 
-		if (in_array($contact['network'], Protocol::FEDERATED)) {
-			$channel_settings_label = $this->t('Channel Settings');
-			$channel_frequency     = Contact\User::getChannelFrequency($contact['id'], $this->session->getLocalUserId());
-		} else {
-			$channel_settings_label = null;
-			$channel_frequency     = null;
-		}
+		$channel_settings_label = $this->t('Channel Settings');
+		$channel_frequency     = Contact\User::getChannelFrequency($contact['id'], $this->session->getLocalUserId());
```

Comment:
Please move the `L10n` call to the template variable array below now that the variable can't be anything else.
```suggestion
		$channel_frequency = Contact\User::getChannelFrequency($contact['id'], $this->session->getLocalUserId());
```

Reply:
I don't understand what you mean here.

Reply:
Please move the `$this->t('Channel Settings')` call to the current line 429 that reads:
```php
'$channel_settings_label' => $channel_settings_label,
```
instead of the variable, that you should remove as it doesn't have any other use now.

Reply:
Done